cypress/e2e/awx/administration/workflowApproval.cy.ts

```diff
@@ -0,0 +1,206 @@
+    cy.deleteAwxUser(userWFCancel, { failOnStatusCode: false });
+    cy.deleteAwxUser(userWFDeny, { failOnStatusCode: false });
+  });
+
+  function refreshToDelete(wfaID: string) {
+    // FIXME: the page needs to be refreshed to button be available
+    cy.visit(`/administration/workflow-approvals/`);
+    cy.filterTableByTextFilter('id', wfaID);
+    cy.getTableRow('id', wfaID).within(() => {
+      cy.getByDataCy('actions-column-cell').within(() => {
+        cy.clickKebabAction('actions-dropdown', 'delete-workflow-approval');
+      });
+    });
+    cy.actionsWFApprovalConfirmModal('delete');
+  }
+
+  it.skip('admin can create a WF approval and assign user with access to approve and then delete the WF from the list toolbar', () => {
+    cy.giveUserWfjtAccess(workflowJobTemplate.name, userWFApprove.id, 'Approve');
+  });
+
+  it.skip('admin can create a WF approval and assign user with access to deny and then delete the WF from the list toolbar', () => {
+    cy.giveUserWfjtAccess(workflowJobTemplate.name, userWFDeny.id, 'Approve');
+  });
+
+  it.skip('admin can create a WF approval and assign user with access to cancel and then delete the WF from the list toolbar', () => {
+    cy.giveUserWfjtAccess(workflowJobTemplate.name, userWFCancel.id, 'Approve');
+  });
+
+  it.skip('admin can enable concurrent jobs in a WFJT with a WF approval node, launch the job multiple times, bulk approve, and then bulk delete from the list toolbar', () => {});
+
+  it.skip('admin can enable concurrent jobs in a WFJT with a WF approval node, launch the job multiple times, bulk deny, and then bulk delete from the list toolbar', () => {});
+
+  it('admin can approve and then delete a workflow approval from the list row item', () => {
+    cy.wait('@launchWFJT')
+      .its('response')
+      .then((response: { body: { workflow_job: number } }) => {
+        cy.getAwxWFApprovalByWorkflowJobID(response.body.workflow_job).then((wfa) => {
+          cy.visit(`/administration/workflow-approvals/`);
+          cy.filterTableByTextFilter('id', wfa.id.toString());
+          cy.getTableRow('id', wfa.id.toString()).within(() => {
+            cy.getByDataCy('actions-column-cell').within(() => {
+              cy.getByDataCy('approve').click();
+            });
+          });
+          cy.actionsWFApprovalConfirmModal('approve').then(() => {
+            // FIXME: the page needs to be refreshed to button be available
+            refreshToDelete(wfa.id.toString());
+          });
+        });
+      });
+  });
+
+  it('admin can deny and then delete a workflow approval from the list row item', () => {
+    cy.wait('@launchWFJT')
+      .its('response')
+      .then((response: { body: { workflow_job: number } }) => {
+        cy.getAwxWFApprovalByWorkflowJobID(response.body.workflow_job).then((wfa) => {
+          cy.visit(`/administration/workflow-approvals/`);
+          cy.filterTableByTextFilter('id', wfa.id.toString());
```

Comment:
I saw that filter is being applied twice. It seems like `getTableRow` also filters by  filterTableByTextFilter

```Cypress.Commands.add(
  'getTableRow',
  (columnDataCy: string, text: string, options?: { disableFilter?: boolean }) => {
    if (options?.disableFilter !== true) {
      cy.filterTableByTextFilter(columnDataCy, text);
    }
    if (columnDataCy === 'id') {
      cy.contains(`[data-cy="row-id-${text}"]`, text).parents('tbody');
    } else {
      cy.contains(`[data-cy="${columnDataCy}-column-cell"]`, text).parents('tr');
    }
  }
);```